experimental/SetPartitions/src/AbstractPartition.jl

```diff
@@ -0,0 +1,26 @@
+"""
+    AbstractPartition
+
+Abstract type for classical-, colored- and spatial partitions.
+"""
+abstract type AbstractPartition end
+
+function composition(p::T, q::T) where {T <: AbstractPartition}
+    return composition_loops(p, q)[1]
+end
+
+function ⋅(p::T, q::T) where {T <: AbstractPartition}
+    return composition(p, q)
+end
```

Comment:
This symbol is used nowhere else in Oscar. And I think having * should be enough, right?